### PR TITLE
Topping

### PR DESCRIPTION
**Pull request by surangap**
### Changelog
- Rename trnnut to topping

### CODE REVIEW

File: src/doughnut/topping/mod.rs [hunk 1]
```diff
@@ -1,8 +1,7 @@
-// Copyright 2022-2023 Futureverse Corporation Limited
+// Copyright 2022-2023 Futureverse Corporation Limited//!
```
Comment (JasonTulp) [inline src/doughnut/topping/mod.rs:1]: Something strange happened with this comment
```suggestion
//! Copyright 2022-2023 Futureverse Corporation Limited
```

File: src/doughnut/v1.rs [hunk 2]
```diff
@@ -30,15 +30,15 @@ use core::convert::TryFrom;
 
 const NOT_BEFORE_MASK: u8 = 0b0000_0001;
 
-const MAX_DOMAINS: usize = 128;
+const MAX_toppingS: usize = 128;
```
Comment (JasonTulp) [inline src/doughnut/v1.rs:33]: nit.
```suggestion
const MAX_TOPPINGS: usize = 128;
```

**Review by JasonTulp — APPROVED** [review]

File: src/doughnut/topping/mod.rs (hunk 1 above)
Comment (zees-dev) [reply src/doughnut/topping/mod.rs:1]: Should also change the copyright from 2023-2024

File: src/doughnut/v1.rs [hunk 3]
```diff
@@ -466,7 +466,7 @@ mod test {
                 issuer:[0_u8; 33],
                 holder:[1_u8; 33],
                 fee_mode: 0,
-                domains:vec![("cennznet".to_string(), vec![0])],
+                toppings:vec![("cennznet".to_string(), vec![0])],
```
Comment (zees-dev) [inline src/doughnut/v1.rs:469]: nit: remove references to cennz

**Review by zees-dev — APPROVED** [review]
LGTM 👍 

Eventually we should also rename the modules - since we have functions like `topping.get_modules`.
The module should also be a concept related to the topping 😅

**Pull request merged**